app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
```

Comment:
you don't need to pass default `appetite` param here, just pass it in `super`

Reply:
what about this?